source/manuals/programming-languages/java.html.md.erb

```diff
@@ -16,13 +16,11 @@ We favour consistency across our code, so when considering using a new or differ
 
 Variable and field names should match the class they are instantiating, or have a descriptive name in the context of their use.
 
-Use the default code indenting in IntelliJ.
-
-Ensure line feeds are present at the ends of files.  This can be enabled as an option in IntelliJ: `Settings->Editor->Other->Ensure line feed at file end on save.`
+Use the [GDS Way EditorConfig file](editorconfig), which has settings for things like code indentation. Place a copy of this file named `.editorconfig` in the root of your project to have IntelliJ IDEA and some other editors automatically apply the settings. If your editor does not support EditorConfig, manually configure its settings to match.
 
 ## Dependency injection (DI)
 
-Some programmes use [Guice](https://github.com/google/guice) as a dependency injection framework, however, consider whether use of DI is appropriate for your project.
+Consider whether dependency injection is appropriate for your project before using it. Some programmes use the [Guice](https://github.com/google/guice) dependency injection framework.
```

Comment:
I think this kind of language is too vague for a styleguide: "consider if X is appropriate"

Is there anything in particular to consider, that you wouldn't have already thought about before reading the styleguide? If not I think we can drop this sentence.

Reply:
I think the desire here was to stop people from blindly using DI without thinking about it

Reply:
Hmm, I'm just skeptical it would actually deter anyone unless we give examples of what is and isn't appropriate.

Reply:
I think I’d have to defer to the people who were most vocal about wanting to put more emphasis on considering the usefulness of DI for examples

Reply:
No problem, I wasn't at the meeting so am missing some context here